resources/views/frontend/trainee-profile/add-guardian-information.blade.php

```diff
@@ -7,13 +7,53 @@
 @extends($layout)
 
 @section('title')
-{{$siteSettingInfo->site_title}} :: {{__('frontend.trainee.guardian_information')}}
+    {{ $siteSettingInfo->site_title }} :: {{ __('frontend.trainee.guardian_information') }}
 @endsection
 
 @push('css')
     <style>
+        .content-wrapper {
+            align-content: center !important;
+        }
+
+        .form-control {
+            background-color: #ffff !important;
+            border: 1px solid #E9EAF0 !important;
+        }
+
+        .input-group-text {
+            background-color: #ffff !important;
+            border: 1px solid #E9EAF0 !important;
+        }
+
+        label {
+            font-weight: normal !important;
+        }
+
+        .back-to-profile {
```

Comment:
`.back-to-profile {
            background-color: #ffffff;
            border-radius: 12px;
            box-shadow: 0px 4px 4px #00000040;
            color: #671688;
            font-weight: 700;
        }
` Change every back to profile button to this